package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "auto-block",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
This would generate an error if they were using something incorrectly but it isn't changing intended functionality.